Just a few enhancments I assume

meysam81 opened the pull request:
Check it out if you like!

meysam81 committed:
Customized set function

meysam81 committed:
Modified git ignore

meysam81 committed:
Auto indent using Qt

meysam81 committed:
Number of elements in a specified row

meysam81 committed:
add removeAnyEdge & removeEdge

uestla commented:
Thanks for your changes.

Could you please maybe separate them into one commit per one PR? Each of your commits has its own functionality/meaning so it would be helpful to be able to discuss those in separate discussion threads.

Thanks!

meysam81 commented:
I checked out and could not find how to to so sir!

meysam81 committed:
add getNeighbors for a specific row

uestla committed:
optimized vector values access - thanks to @sergiosvieira [Closes #14]

This replaces calling std::vector::at() with direct operator[] access.
It is somehow faster - maybe compiler-related optimization or lack of out-of-range checking when using operator[].
Out-of-range access is prevented thanks to validateCoordinates() method

uestla committed:
tests: better folder structure

uestla committed:
tests: separated helper functions into standalone file

uestla committed:
tests: moved standard operations into functions

uestla commented:
@meysam81 Ideally each functionality/logical change should have its own PR, hence its own branch. Reviewing and discussion will be more effective and well arranged.

meysam81 committed:
Merge branch 'master' into devel-5

meysam81 committed:
merge from uestla/master to uestla-master

meysam81 committed:
Merge branch 'uestla-master'

meysam81 committed:
Merge branch 'master' into devel-5

meysam81 committed:
resolving merge conflicts from the last merge

what have I done in the past??

meysam81 committed:
Merge branch 'master' into devel-5

meysam81 committed:
Merge pull request #2 from meysam81/devel-5

Devel 5

meysam81 committed:
checkout to commit 6f34e23
